ocs_ci/utility/kms.py

```diff
@@ -178,6 +179,26 @@ def vault_namespace_exists(self, vault_namespace):
                 return False
         return True
 
+    def vault_backend_path_exists(self, backend_path):
+        """
+        Check if vault backend path already exists
+
+        Args:
+            backend_path (str): name of the vault backend path
+
+        Returns:
+            bool: True if exists else False
+
+        """
+        cmd = f"vault secrets list | grep {backend_path}"
```

Comment:
Instead of grep can we use json output from vault commands, which would be more easy to parse and for future use as well.

Reply:
Done